src/screens/People/People.js

```diff
@@ -681,15 +629,35 @@ class PeopleScreen extends React.Component<Props, State> {
     const {
       localContacts,
       chats,
+      theme,
+      goToInvitationFlow,
     } = this.props;
     const inSearchMode = query.length >= MIN_QUERY_LENGTH;
 
     const sortedLocalContacts = sortLocalContacts(localContacts, chats);
     const contact = sortedLocalContacts.find(({ id }) => id === manageContactId);
 
+    const colors = getThemeColors(theme);
+
     return (
       <ContainerWithHeader
-        headerProps={{ noBack: true, leftItems: [{ title: 'People' }] }}
+        headerProps={{
+          noBack: true,
+          leftItems: [{ title: 'People' }],
+          rightItems: [{
+            custom: (
+              <IconButton
+                icon="present"
+                color={colors.positive}
+                iconText="Invite friends"
+                style={{ flexDirection: 'row' }}
```

Comment:
Oh and one more tiny thing: 
seems like horizontal alignment is a tiny bit off - this button should to be 2px or so lower to be horizontally centered with screen's title